src/apps/bridge/bridgePowerController.cpp

```diff
@@ -312,13 +312,12 @@ void BridgePowerController::powerControllerRun(void *arg) {
 
 bool BridgePowerController::getAdinDevice() {
   bool rval = false;
-  for (uint32_t port = 0; port < bm_l2_get_num_ports(); port++) {
+  for (uint32_t port = 0; port < bm_l2_get_num_devices(); port++) {
```

Comment:
This API had to be changed because it was not accurrate, we are getting the handle for a device not the handle for a specific port.

Reply:
👍, just putting down comment from meeting to switch the "port" variable to "device" or something more accurate

Reply:
Changed!